layers/+lang/python/packages.el

```diff
@@ -105,6 +105,15 @@
         "hh" 'anaconda-mode-show-doc
         "gu" 'anaconda-mode-find-references))))
 
+(defun python/post-init-window-purpose ()
+  (add-hook 'python-mode-hook (lambda ()
+                                (spacemacs|window-purpose-define-config
+                                 python (:name-purposes '(("*compilation*" . compile))
+                                                        :regexp-purposes '(("^\*Pydoc" . help)
+                                                                           ("^\*Pylookup" . help)
+                                                                           ("^\*Anaconda" . help)
+                                                                           ("^\*live-py" . compile)))))))
```

Comment:
`compile` and `help` are not good purpose names, because there are also functions with the same names (this causes problems if configuring `purpose-special-action-sequences`). Use `Compile` and `Help` instead

Reply:
Hmm you are right, you have mentioned that in the original post ... I will fix it.

Reply:
Done